app/model/Forms.scala

```diff
@@ -143,8 +143,10 @@ object Forms {
   object WhitelistedIpForm {
     private val privateNetworkRanges = Set(
```

Comment:
This is good but I think we should change the name now as these are not all private networks anymore.

Reply:
done - see latest commit